### PR TITLE
Fix blank pages in GET routes

### PR DESCRIPTION
**Pull request by SaphiLC**
Hope this fixes the problem, it did for me in my local server :)

### CODE REVIEW

File: src/ProtectAgainstSpam.php [hunk 1]
```diff
@@ -19,7 +19,7 @@ public function __construct(SpamResponder $spamResponder)
 
     public function handle(Request $request, Closure $next): Response
     {
-        if (! config('honeypot.enabled')) {
+        if (! config('honeypot.enabled') || $request->method() === 'GET') {
```
Comment (michgeek) [inline src/ProtectAgainstSpam.php]: If it's only on `POST` request, should this be `$request->method() !== 'POST'` instead?

Comment (freekmurze) [reply src/ProtectAgainstSpam.php]: Yup

**Comment by freekmurze** [comment]
Could you also add a test that proves that the problem is solved?

**Comment by SaphiLC** [comment]
What about DELETE, UPDATE, etc, methods then? Also, no idea how to make a test, sorry

**Comment by michgeek** [comment]
@SaphiLC You could add it in the `tests/ProtectionAgainstSpam.php`

See https://github.com/spatie/laravel-honeypot/pull/14#pullrequestreview-201419654 for an example of test

**Comment by freekmurze** [comment]
This has been fixed in the latest version.

**Pull request closed without merging**